modules/rpc/thrift_test.go

```diff
@@ -74,6 +76,11 @@ modules:
 
 	testInitRunModule(t, goofy[0], mci)
 	testInitRunModule(t, gopher[0], mci)
+
+	// Dispatcher must be resolved in the default graph
```

Comment:
Why does FX need a default graph at all? Why not just instantiate 1 graph when FX is instantiated?